### PR TITLE
chore(ru): handling redirects of chrome.com

### PR DESCRIPTION
**Pull request by PassionPenguin**
1. handling redircts of `chrome.com`

### CODE REVIEW

File: files/ru/glossary/developer_tools/index.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ slug: Glossary/Developer_Tools
 
 - [Firefox Developer Tools](/ru/docs/Tools) on MDN
 - [Firebug](https://getfirebug.com/) (former developer tool for Firefox)
-- [Chrome DevTools](https://developer.chrome.com/devtools) on chrome.com
+- [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools?hl=ru) on chrome.com
```
Comment (PassionPenguin) [inline files/ru/glossary/developer_tools/index.md:16]:
```suggestion
- [Chrome DevTools](https://developer.chrome.com/docs/devtools/?hl=ru) on chrome.com
```

File: files/ru/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ slug: Learn/Common_questions/Tools_and_setup/What_are_browser_developer_tools
 
 - [Firefox Page inspector](/ru/docs/Tools/Page_Inspector)
 - [IE DOM Explorer](http://msdn.microsoft.com/en-us/library/ie/dn255008%28v=vs.85%29.aspx)
-- [Chrome DOM inspector](https://developer.chrome.com/devtools/docs/dom-and-styles) (Inspector в Opera схож с Inspector в Chrome)
+- [Chrome DOM inspector](https://developers.google.com/web/tools/chrome-devtools/?hl=ru) (Inspector в Opera схож с Inspector в Chrome)
```
Comment (PassionPenguin) [inline files/ru/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md:89]:
```suggestion
- [Chrome DOM inspector](https://developer.chrome.com/docs/devtools/?hl=ru) (Inspector в Opera схож с Inspector в Chrome)
```

File: files/ru/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md [hunk 3]
```diff
@@ -141,5 +141,5 @@ slug: Learn/Common_questions/Tools_and_setup/What_are_browser_developer_tools
 
 - [Firefox Web Console](/ru/docs/Tools/Web_Console)
 - [IE JavaScript console](http://msdn.microsoft.com/en-us/library/ie/dn255006%28v=vs.85%29.aspx)
-- [Chrome JavaScript Console](https://developer.chrome.com/devtools/docs/console) (Inpector в Opera схож с Inspector в Chrome)
+- [Chrome JavaScript Console](https://developers.google.com/web/tools/chrome-devtools/?hl=ru) (Inpector в Opera схож с Inspector в Chrome)
```
Comment (PassionPenguin) [inline files/ru/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md:144]:
```suggestion
- [Chrome JavaScript Console](https://developer.chrome.com/docs/devtools/?hl=ru) (Inpector в Opera схож с Inspector в Chrome)
```

File: files/ru/web/api/console/clear_static/index.md [hunk 4]
```diff
@@ -29,4 +29,4 @@ console.clear();
 
 - [Opera Dragonfly documentation: Console](http://www.opera.com/dragonfly/documentation/console/)
 - [MSDN: Using the F12 Tools Console to View Errors and Status](http://msdn.microsoft.com/library/gg589530)
-- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/devtools/docs/console#assertions)
+- [Chrome Developer Tools: Using the Console](https://developers.google.com/web/tools/chrome-devtools/?hl=ru#assertions)
```
Comment (PassionPenguin) [inline files/ru/web/api/console/clear_static/index.md:32]:
```suggestion
- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/docs/devtools/?hl=ru#assertions)
```

File: files/ru/web/api/console/clear_static/index.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ slug: Web/API/console/clear_static
 
 Содержимое консоли заменяется на информационное сообщение, которое может быть следующим: "Консоль очищена".
 
-Примечание для Google Chrome, от функции `console.clear()` не будет никакого эффекта, если у пользователя установлена надстройка "Preserve log upon navigation" в [настройках](https://developer.chrome.com/devtools/docs/settings#preserve-log%20upon%20navigation).
+Примечание для Google Chrome, от функции `console.clear()` не будет никакого эффекта, если у пользователя установлена надстройка "Preserve log upon navigation" в [настройках](https://developers.google.com/web/tools/chrome-devtools/?hl=ru#preserve-log%20upon%20navigation).
```
Comment (PassionPenguin) [inline files/ru/web/api/console/clear_static/index.md:12]:
```suggestion
Примечание для Google Chrome, от функции `console.clear()` не будет никакого эффекта, если у пользователя установлена надстройка "Preserve log upon navigation" в [настройках](https://developer.chrome.com/docs/devtools/?hl=ru#preserve-log%20upon%20navigation).
```

File: files/ru/web/api/console/assert_static/index.md [hunk 6]
```diff
@@ -86,4 +86,4 @@ console.assert(false, "слово %s", "foo");
 
 - [Opera Dragonfly documentation: Console](http://www.opera.com/dragonfly/documentation/console/)
 - [MSDN: Using the F12 Tools Console to View Errors and Status](http://msdn.microsoft.com/library/gg589530)
-- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/devtools/docs/console#assertions)
+- [Chrome Developer Tools: Using the Console](https://developers.google.com/web/tools/chrome-devtools/?hl=ru#assertions)
```
Comment (PassionPenguin) [inline files/ru/web/api/console/assert_static/index.md:89]:
```suggestion
- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/docs/devtools/?hl=ru#assertions)
```

File: files/ru/web/api/console/index.md [hunk 7]
```diff
@@ -64,7 +64,7 @@ console.log("Failed to open the specified link");
 - {{domxref("console.timeEnd()")}}
   - : Останавливает указанный [таймер](/ru/docs/Web/API/console#Timers) и записывает в лог, сколько прошло секунд от его старта.
 - {{domxref("console.timeStamp()")}} {{Non-standard_inline}}
-  - : Ставит маркер в браузерном инструменте измерения производительности [Timeline](https://developer.chrome.com/devtools/docs/timeline) или [Waterfall](/ru/docs/Tools/Производительность/Waterfall).
+  - : Ставит маркер в браузерном инструменте измерения производительности [Timeline](https://developers.google.com/web/tools/chrome-devtools/?hl=ru) или [Waterfall](/ru/docs/Tools/Производительность/Waterfall).
```
Comment (PassionPenguin) [inline files/ru/web/api/console/index.md:67]:
```suggestion
  - : Ставит маркер в браузерном инструменте измерения производительности [Timeline](https://developer.chrome.com/docs/devtools/?hl=ru) или [Waterfall](/ru/docs/Tools/Производительность/Waterfall).
```

**Comment by PassionPenguin** [comment]
> [!WARNING]
> Entries related to chrome devtools may need a thorough cleanup. Links pointing to chrome devtools have a tremendous change and such documents on google developer site are no longer available.

**Review by leon-win — APPROVED** [review]

**Pull request merged**